### ISSUE TITLE
Error save project

### ISSUE THREAD

**Issue by willianholtz** [deepnight/ldtk#484]
### Error saving any type of project.
 latest version, linux.

![sav erro 2](https://user-images.githubusercontent.com/23488207/115943636-6bbcaf00-a487-11eb-9069-75dae4162beb.png)
![erro 3](https://user-images.githubusercontent.com/23488207/115943634-6a8b8200-a487-11eb-9aaa-7f3da0935a06.png)
![save error](https://user-images.githubusercontent.com/23488207/115943637-6bbcaf00-a487-11eb-9c4b-82810f9ed8dc.png)

**Comment by deepnight** [comment]
Thanks for reporting! Could you please copy/paste the "Error infos" here too?

**Comment by willianholtz** [comment]
`
Stack:
```
LDtk version: 0.9.3-64bits
ENOTDIR: not a directory, mkdir '/home/willian/Project89/assets/obj_maps/asds/png'
Error
Error: ENOTDIR: not a directory, mkdir '/home/willian/Project89/assets/obj_maps/asds/png'
    at Object.mkdirSync (fs.js:845:3)
    at Function.dn_js_NodeTools.createDirs (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:61417:15)
    at ui_ProjectSaving.initDir (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179375:20)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179109:10)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179077:10)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179026:9)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:178947:9)
    at file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:178873:13
    at page_Editor.checkAutoLayersCache (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:152090:4)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:178872:20)
```
Processes:
```
#0 App
 |--- #2 page.Editor
       |--- #3 misc.FileWatcher
       |--- #4 display.WorldRender
       |--- #5 display.LevelRender
       |--- #6 display.Camera
       |--- #7 display.Rulers[--]
       |--- #8 tool.SelectionTool[--]
       |--- #9 tool.lt.DoNothing[--]
       |--- #10 WorldTool
       |--- #11 tool.PanView[--]
       |--- #12 tool.lt.TileTool[--]
       |--- #16 ui.modal.panel.EditProject
       |--- #18 ui.Tip
       |--- #19 ui.ProjectSaving
```

Log:
```
[FILE]         Cleared all file watches
[EVENT]        LayerInstanceSelected
[FILE]         Reloading tileset: TilesetBase path=TilesetBase.png
[FILE]         Loading atlas image: TilesetBase.png...
[CACHE]        Caching image TilesetBase.png...
[CACHE]         -> identified as Png
[FILE]          -> Loaded 98157 bytes.
[FILE]          -> Decoded 768x256 pixels.
[GENERAL]      Tileset remapping...
[FILE]          -> Reload result: Ok
[FILE]          -> pixelData: Ok
[EVENT]        TilesetDefChanged 2
[FILE]         Reloading tileset: S4m_ur4i_metroidvania_1_3_high_contrast path=s4m_ur4i-metroidvania-1.3-high-contrast.png
[FILE]         Loading atlas image: s4m_ur4i-metroidvania-1.3-high-contrast.png...
[CACHE]        Caching image s4m_ur4i-metroidvania-1.3-high-contrast.png...
[CACHE]         -> identified as Png
[FILE]          -> Loaded 37766 bytes.
[FILE]          -> Decoded 256x448 pixels.
[GENERAL]      Tileset remapping...
[FILE]          -> Reload result: Ok
[FILE]          -> pixelData: Ok
[EVENT]        TilesetDefChanged 12
[EVENT]        ProjectSelected
[RENDER]       Reset world render
[FILE]         Watching file: /home/willian/Project89/assets/obj_maps/TilesetBase.png
[FILE]         Watching file: /home/willian/Project89/assets/obj_maps/s4m_ur4i-metroidvania-1.3-high-contrast.png
[RENDER]       Rendered world level data.Level.Level1(#0)
[RENDER]       Rendered level UI
[RENDER]       Rendered world level data.Level.Level_1(#20)
[FILE]         Loading HTML template editProject
[FILE]         Remap project paths: /home/willian/Project89/assets/obj_maps => /home/willian/Project89/assets/obj_maps
[FILE]         Remap file path: TilesetBase.png => TilesetBase.png
[FILE]         Remap file path: s4m_ur4i-metroidvania-1.3-high-contrast.png => s4m_ur4i-metroidvania-1.3-high-contrast.png
[SAVE]         [asds] Preparing project saving...
[SAVE]         [asds] => PreChecks...
[SAVE]         [asds] => BeforeSavingActions...
[EVENT]        BeforeProjectSaving
[SAVE]         [asds] => AutoLayers...
[SAVE]         [asds] => SavingMainFile...
[SAVE]         [asds]   Preparing SavingData...
[SAVE]         [asds]   Writing /home/willian/Project89/assets/obj_maps/asds...
[SAVE]         [asds] => SavingLayerImages...
[GENERAL]      Page started: page.CrashReport()
[FILE]         Loading page template: crashReport from /usr/share/ldtk/resources/app.asar/assets/tpl/pages/crashReport.html
[ERROR]        ENOTDIR: not a directory, mkdir '/home/willian/Project89/assets/obj_maps/asds/png' (Error)
[ERROR]        Error: ENOTDIR: not a directory, mkdir '/home/willian/Project89/assets/obj_maps/asds/png'
    at Object.mkdirSync (fs.js:845:3)
    at Function.dn_js_NodeTools.createDirs (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:61417:15)
    at ui_ProjectSaving.initDir (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179375:20)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179109:10)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179077:10)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:179026:9)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:178947:9)
    at file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:178873:13
    at page_Editor.checkAutoLayersCache (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:152090:4)
    at ui_ProjectSaving.beginState (file:///usr/share/ldtk/resources/app.asar/assets/js/renderer.js:178872:20)

[GENERAL]      
#0 App
 |--- #2 page.Editor
       |--- #3 misc.FileWatcher
       |--- #4 display.WorldRender
       |--- #5 display.LevelRender
       |--- #6 display.Camera
       |--- #7 display.Rulers[--]
       |--- #8 tool.SelectionTool[--]
       |--- #9 tool.lt.DoNothing[--]
       |--- #10 WorldTool
       |--- #11 tool.PanView[--]
       |--- #12 tool.lt.TileTool[--]
       |--- #16 ui.modal.panel.EditProject
       |--- #18 ui.Tip
       |--- #19 ui.ProjectSaving
 |--- #20 page.CrashReport

[FILE]         Cleared all file watches
````

**Comment by deepnight** [comment]
This should be fixed in 0.10.0
